frontend/app/services/SubscriptionService.scala

```diff
@@ -155,6 +156,17 @@ class SubscriptionService(val tierPlanRateIds: Map[ProductRatePlan, String],
     } yield subscriptionDetails
   }
 
+  def getUsageCountWithinTerm(memberId: MemberId, unitOfMeasure: String): Future[Int] = (for {
+    account <- getAccount(memberId)
+    subscriptions <- zuoraRestService.subscriptionsByAccount(account.id)
+    subscription = subscriptions.head if subscriptions.nonEmpty
+    startDate = formatDateTime(subscription.termStartDate)
+    whereClause = s"StartDateTime >= '$startDate' AND SubscriptionID = '${subscription.id}'"
```

Comment:
As it is, this methods depends on the `ZuoraRestService` in order to fetch the subscription record. I am not really keen on moving this into the SoapClient and ending up with a soap client that somehow depends on the rest client...